### PR TITLE
New package: shawnjb.touchforwindows version 1.0.0

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123549)

### CODE REVIEW

**Comment by shawnjb** [comment]
@microsoft-github-policy-service agree [company="shawnjb"]

**Comment by shawnjb** [comment]
@microsoft-github-policy-service agree

**Comment by shawnjb** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

**Comment by stephengillie** [comment]
Hi @shawnjb,

Does this package have a framework or other dependency?

**Comment by shawnjb** [comment]
> Hi @shawnjb,
> 
> Does this package have a framework or other dependency?

The dependancies are listed below:
- Standard C/C++ Libraries
- Microsoft Visual C++ Redistributable
  - `Microsoft.VCRedist.2015+.x64`
- File System Access
- Registry Access

**Comment by stephengillie** [comment]
Thanks for the update. Does `ucrtbased.dll` come from any of those frameworks?

**Comment by shawnjb** [comment]
> ucrtbased.dll

ucrtbased.dll is part of the Universal C Runtime (UCRT) library in Microsoft Windows. The "ucrtbased.dll" error indicates a missing UCRT version. Resolve it by installing the matching Microsoft Visual C++ Redistributable Package.

File: manifests/s/shawnjb/touchforwindows/1.0.0/shawnjb.touchforwindows.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: shawnjb.touchforwindows
+PackageVersion: 1.0.0
+InstallerType: inno
```
Comment (stephengillie) [inline manifests/s/shawnjb/touchforwindows/1.0.0/shawnjb.touchforwindows.installer.yaml:6]:
```suggestion
InstallerType: inno
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by stephengillie** [comment]
> ucrtbased.dll is part of the Universal C Runtime (UCRT) library in Microsoft Windows.
> 
> It should be in System32: image

Ah, that framework is missing from the Manual Validation pipeline:
image
And it needs a manifest added to WinGet, because it hasn't been yet.

**Comment by stephengillie** [comment]
This appears to be the UCRT package location: https://www.microsoft.com/en-us/download/details.aspx?id=48234

**Comment by shawnjb** [comment]
> This appears to be the UCRT package location: https://www.microsoft.com/en-us/download/details.aspx?id=48234

I'll test about with this in Windows Sandbox and see if one of the redist packages contains this compatability.

**Comment by shawnjb** [comment]
This may help, apparently it comes with Visual C++ https://learn.microsoft.com/en-us/cpp/c-runtime-library/crt-library-features?view=msvc-160#c-runtime-lib-files

**Comment by shawnjb** [comment]
I see the problem now, I am using a debug version of the DLL.

**Comment by shawnjb** [comment]
My mistake was forgetting to add the Release flag on build args, I will fix this.

**Comment by shawnjb** [comment]
The file was just overridden with a release build, is there anything else I need to do?

**Pull request closed without merging**